alert-city/client
# Feat: Theme setting

## TL;DR
- Added Mui Theme to handle theme change
- Added a setting icon at the login page to set theme and language
- Optimized the switchTheme.ts
- Added themeColour.ts and  ThemeWrapper.tsx into /modules/theme
- Adjusted some components to fit the dark mode

## How Has This Been Tested?
1. Click the setting icon at the login page and change the theme, then see the effect
2. Login  and click the setting icon at the top bar  and change the theme, then see the effect

## Screenshots
<img width="1697" alt="image" src="https://github.com/user-attachments/assets/85d10056-654f-4875-93ff-984c29917667">

<img width="1697" alt="image" src="https://github.com/user-attachments/assets/75473679-34b1-4a5a-9b73-54aa4496a9b9">

<img width="1697" alt="image" src="https://github.com/user-attachments/assets/386740bc-90ac-4fff-aecf-6e5ecd56169d">

<img width="1697" alt="image" src="https://github.com/user-attachments/assets/c18729a0-bec1-4c2a-8389-38b6505a140a">

<img width="1697" alt="image" src="https://github.com/user-attachments/assets/4e9ef3f7-97ad-4f5e-9d9d-1be6cac28835">


## Resolve 
Resolve CP-31
